source/lac/trilinos_epetra_communication_pattern.cc

```diff
@@ -45,7 +45,7 @@ namespace LinearAlgebra
       comm = std_cxx11::make_shared<const MPI_Comm>(communicator);
 
       Epetra_Map vector_space_vector_map = vector_space_vector_index_set.make_trilinos_map(*comm,
-                                           false);
+                                           true);
```

Comment:
Why did you change this? VectorSpaceVector are not supposed to have overlapping dofs because this won't be allowed in Tpetra